docs/src/UserGuide/faq.md

```diff
@@ -77,47 +77,127 @@ Now we can concatenate `ds1` and `ds2`:
 dsfinal = concatenatecubes([ds1, ds2], Dim{:Variables}(["var1", "var2"]))
 ````
 
-## How do I subset a Cube?
+## How do I subset a YAXArray, Cube or Dataset?
```

Comment:
```suggestion
## How do I subset a YAXArray ( Cube ) or Dataset?
```
a cube is a yaxarray.

Reply:
see: https://juliadatacubes.github.io/YAXArrays.jl/dev/UserGuide/types#(Data)-Cube